elasticmagic/document.py

```diff
@@ -141,6 +141,7 @@ def __init__(self, _hit=None, _result=None, **kwargs):
         self._index = self._type = self._id = self._score = None
         self._hit_fields = None
         self._highlight = None
+        self.matched_queries = None
```

Comment:
Think field should be private and there should be method like `get_matched_queries` or `get_matched_query_names`.